lib/kitchen/verifier/inspec.rb

```diff
@@ -36,12 +36,24 @@ class Inspec < Kitchen::Verifier::Base
 
       default_config :inspec_tests, []
 
+      def add_target(runner, target, opts)
```

Comment:
I guess this https://github.com/chef/kitchen-inspec/pull/74/files#diff-873cf55477008bb3a2a80b3fa94e7a06R42 is the reason, the "profile metadata workaround". @chris-rock are you proposing a different workaround?